docs/integrations/feeds.md

```diff
@@ -154,7 +154,12 @@ Let's review the object key and values.
         URL = "URL"
     ```
 * `"rawJSON"` - _required_. This dictionary should contain the `"value"` and `"type"` fields as well as any other unmodified data returned from the feed source about an indicator.
-* `"fields"` - _optional_. A dictionary that maps values to existing indicator fields defined in Cortex XSOAR where the key is the `cliname` of an indicator field.
+* `"fields"` - _optional_. A dictionary that maps values to existing indicator fields defined in Cortex XSOAR where the key is the `cliname` of an indicator field. 
+   To see the full list of possible fields:
+   * Go to the `Settings` section in Cortex XSOAR.
+   * Click the `ADVANCED` tab. 
+   * Go to the `Fields` section and filter `indicator` fields.
+   * To inspect a specific field - tick the box near the field name and click `Edit.
```

Comment:
Mention that the `cliname` is the `machinename` when inspecting the indicator field in the UI.